net-im/skypeforlinux/Manifest

```diff
@@ -1,2 +1,7 @@
 DIST skypeforlinux_8.90.0.407-1.x86_64.rpm 126406385 BLAKE2B ef3627e786b60a512576a6cee5af8b53c10a54122cb2de784a534c5effb13ea053070796a52c6672344dc5abb6dc3b503e89a1f14960c8990173b21acfc0b8e2 SHA512 ad74e6e8006f23c2f70f67373fa386e19e3b9e4da5e1ab29956191bb3626099d06ebcc80ace458401dd4f4a62d894eb735f7be0a60b6f668d576aaadf119d39d
 DIST skypeforlinux_8.91.0.404-1.x86_64.rpm 126481575 BLAKE2B 36123aa02ce6099908a7abd0d6c79ea61220339d37613588a251e0f53c605f2aba7c23d653a8f06b17d5d78323dcc3228ac3b5ac0fb90489cd318f2a1d82ebbb SHA512 0199db11aee8a01dcc2e137f2deaf6600caa92d75b113ba65c2869eeda705c3be0e4098c1697c50a75699979ed65389e03befd9de31dc7fb9225452278212c11
+DIST skypeforlinux_8.92.0.401-1.x86_64.rpm 126645771 BLAKE2B 1b1fb1fa3168478e80f772deb49963a50d56c0b2dff41e2c92e3ccc19876a14239ffde662afe382501ba5b7cef162362cbcfc459bc789a805b312e47ce39948d SHA512 cddf41ce5875f4d9efcd4074bda8ade71f0f43c7f912c2c1b5358e2278f8f04b6a8161926ddb1699b076fca493cf71c91b298bcb43d3664d2da7fc35a65eede2
+EBUILD skypeforlinux-8.90.0.407.ebuild 3250 BLAKE2B eed5437754666a9da47304ab1da0539bb8e3e0e7f7e462b7e0de527038cc024f7f0d3c513ffcaaacac9d4c3223de6aa07b26f14f00d2bfa5b9ee872857909eb2 SHA512 09758ed08729dbb173e6d03fc0dbc1e04f3271ba42ff1e837d73bddb88a07a55fe67d64bfd59986a3af67e39d11e170459a2e92e1cf51f7f911cbc3f6b3bf55e
+EBUILD skypeforlinux-8.91.0.404.ebuild 3250 BLAKE2B eed5437754666a9da47304ab1da0539bb8e3e0e7f7e462b7e0de527038cc024f7f0d3c513ffcaaacac9d4c3223de6aa07b26f14f00d2bfa5b9ee872857909eb2 SHA512 09758ed08729dbb173e6d03fc0dbc1e04f3271ba42ff1e837d73bddb88a07a55fe67d64bfd59986a3af67e39d11e170459a2e92e1cf51f7f911cbc3f6b3bf55e
+EBUILD skypeforlinux-8.92.0.401.ebuild 3250 BLAKE2B eed5437754666a9da47304ab1da0539bb8e3e0e7f7e462b7e0de527038cc024f7f0d3c513ffcaaacac9d4c3223de6aa07b26f14f00d2bfa5b9ee872857909eb2 SHA512 09758ed08729dbb173e6d03fc0dbc1e04f3271ba42ff1e837d73bddb88a07a55fe67d64bfd59986a3af67e39d11e170459a2e92e1cf51f7f911cbc3f6b3bf55e
+MISC metadata.xml 461 BLAKE2B 9bcbfe1d6892c40ce917a76c600baad338b5b86967e5656f79b9bf29e75c3ba474dc11444d8e347769f37934610a971097f96b22ae171ea46cd3f1aef2e9f3e5 SHA512 81df3ef175eafabad357f3d5792c71534b85ffd2c1bd9280ea7ef9cbea17aceb396bc7ac65abdb09e5fba8c3d98384c0d67c58a8daa2d42a657e1a7b150b52e7
```

Comment:
How did these get here? You need [thin manifests] for gentoo.git, and I'm not sure if `pkgdev manifest` would even emit this.

Reply:
Oh okay - I'll generate a thin manifest when I get back later today. Thanks ajak